### ISSUE TITLE
A11y_Azure Data Studio extension 'Azure SQL Migration'_Network file share_ScreenReader: In network share path field, backslash(\) present in placeholder text is read twice.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/azuredatastudio#19868]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### Github Tags:
#A11yMAS; #A11yTCS; #A11ySev3; #Benchmark; #SH-AzureSQLMigration-Win32-Jun2022; #DesktopApp; #Win32; #SQL Migration Control Plane; #WCAG1.3.1; #ScreenReader; #Narrator; #NVDA

### Environment and OS details:
Azure Data Studio Version 1.37.0(User Setup)
Windows Version 21H2 (OS Build 22000.675)

### Note:
This issue is not observed with NVDA

### Repro Steps:

1. Turn on Narrator/NVDA
2. Install Azure data studio app and install Azure SQL migration extension
3. Connect to SQL server, tab till respective server, right click and select manage
4. Tab till Azure SQL Migration in sitemap and press enter key
5. Tab till migrate to azure sql and press enter
6. Navigate to databases grid in "Databases for assessment", select few databases and click next.
7. Tab till Network share path field in Database backup step for "My database backups are on a network share"
8. Verify that backslash(\) present in placeholder text is read twice.

### Actual Result:
In network share path field, backslash(\) present in placeholder text is read twice.

### Expected Result:
In network share path field, backslash(\) present in placeholder text should read only once

### Attachment:
![Narrator_backslash](https://user-images.githubusercontent.com/83570638/176434432-fb7d51ed-e55c-498c-b85e-41126c37f814.png)
![NVDA_backslash](https://user-images.githubusercontent.com/83570638/176434493-573904fd-5501-44f4-9536-6f637c67edd9.png)

**Comment by AkshayMata** [comment]
need to fix NETWORK_SHARE_PATH_FORMAT. used \\\\ to escape in string, but narrator is reading without "escaping"

**Comment by AkshayMata** [comment]
our string constant is defined as:

export const NETWORK_SHARE_PATH_FORMAT = localize('sql.migration.network.share.path.format', "\\\\Servername.domainname.com\\Backupfolder");

the placeholder text is properly being escaped, but this is leading to narrator reading \ out twice. @aasimkhan30 will ask to figure out how we can escape to satisfy those conditions

**Comment by aasimkhan30** [comment]
@NiharikaTipirishetty, this is a narrator bug. We do not optimize for individual screen readers. Please close this issue.

**Comment by VijayaBharathyA** [comment]
Here in the Help text backslash () provided two times. Please find attachment for reference.
![MicrosoftTeams-image (3)](https://user-images.githubusercontent.com/100189407/183808383-e747a5eb-8344-442e-9134-ecffecbf56cb.png)

**Comment by VijayaBharathyA** [comment]
Could you please provide Sample app to log an external bug to Narrator?

**Comment by aasimkhan30** [comment]
The Chromium version 98 that is used in ads has an issue causing this bug. The latest one 106 does not have this issue. We can keep it open until ADS gets updated to that version.
